[2022-07-28 15:09:44,289] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: TSLA_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:44,340] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: TSLA_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:44,342] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:44,344] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-28 15:09:44,346] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:44,452] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_historic_table> on 2022-07-28 14:30:00+00:00
[2022-07-28 15:09:44,468] {standard_task_runner.py:52} INFO - Started process 823 to run task
[2022-07-28 15:09:44,485] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'TSLA_historic_data', 'create_historic_table', 'scheduled__2022-07-28T14:30:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/open_options.py', '--cfg-path', '/tmp/tmpwthihbal', '--error-file', '/tmp/tmp76nyna1_']
[2022-07-28 15:09:44,489] {standard_task_runner.py:80} INFO - Job 111: Subtask create_historic_table
[2022-07-28 15:09:44,717] {task_command.py:371} INFO - Running <TaskInstance: TSLA_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [running]> on host a2299eb83309
[2022-07-28 15:09:45,357] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vardhan
AIRFLOW_CTX_DAG_ID=TSLA_historic_data
AIRFLOW_CTX_TASK_ID=create_historic_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T14:30:00+00:00
[2022-07-28 15:09:45,380] {base.py:68} INFO - Using connection ID 'postgres_optionsdata' for task execution.
[2022-07-28 15:09:45,390] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS historic__TSLA_ (
            datetime BIGINT,
            high DOUBLE PRECISION,
            low DOUBLE PRECISION,
            open DOUBLE PRECISION,
            close DOUBLE PRECISION,
            volume INTEGER,
            symbol VARCHAR(32) NOT NULL,
            PRIMARY KEY (symbol, datetime)
        )
    , parameters: None
[2022-07-28 15:09:45,403] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 15:09:45,432] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=TSLA_historic_data, task_id=create_historic_table, execution_date=20220728T143000, start_date=20220728T150944, end_date=20220728T150945
[2022-07-28 15:09:45,516] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 15:09:45,933] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
